charts/k8s-monitoring/ci/ci-integrations-values.yaml

```diff
@@ -49,6 +49,8 @@ metrics:
     enabled: false
   kepler:
     enabled: true
+  beyla:
+    enabled: true
```

Comment:
Is it possible to have a CI test case for this? Like check for some kind of beyla build info metric?

Reply:
Just added that!